packages/@aws-cdk/aws-eks/lib/kubectl-handler/helm/__init__.py

```diff
@@ -82,7 +82,7 @@ def helm_handler(event, context):
 
         if repository is not None and repository.startswith('oci://'):
             tmpdir = tempfile.TemporaryDirectory()
-            chart_dir = get_chart_from_oci(tmpdir.name, release, repository, version)
+            chart_dir = get_chart_from_oci(tmpdir.name, repository, version)
```

Comment:
It's worth mentioning exactly why this does what it does, for those less familiar with helm and its specifics. Can you put a comment above this line explaining why we need the `repository` here?

Reply:
I put extensive explanation above the `rpartition` line. This will be "good enough"?